File: Week-02/Day-1/draw_square.py
# Write a program that reads a number from the standard input, then draws a
# square like this:
#
#
# %%%%%
# %   %
# %   %
# %   %
# %   %
# %%%%%
#
# The square should have as many lines as the number was
nol = int(input('please add a number: '))
def drawer():
    for i in range(1,nol+1):
        if i == 1 or i== nol:
            print("%"*(nol-1))
        else:
            print("%"+" "*(nol-3)+"%")
drawer()

File: Week-07/Day-1/draw-square.js
'use strict';

var lineCount = 8;

// Write a program that draws a
// square like this:
//
//
// %%%%%
// %   %
// %   %
// %   %
// %   %
// %%%%%
//
// The square should have as many lines as lineCount is
console.log('%'.repeat(lineCount/2 + 2));
for (var i = 1; i <=  lineCount - 2; i ++){
    console.log('%' + ' '.repeat(lineCount/2) + '%');
} console.log('%'.repeat(lineCount/2 + 2));